varats/varats/plots/blame_interaction_degree.py

```diff
@@ -652,16 +652,40 @@ def _build_sankey_figure(
     return fig
 
 
+def _get_verified_commit_hash(
+    df: pd.DataFrame, show_interactions_of_commit: str
+) -> str:
+    unique_base_hashes = list(
+        np.unique([str(base_hash) for base_hash in df.base_hash])
+    )
+    unique_inter_hashes = list(
+        np.unique([str(inter_hash) for inter_hash in df.inter_hash])
+    )
+    all_unique_hashes = list(
+        np.unique(unique_base_hashes + unique_inter_hashes)
+    )
+
+    return _get_verified_revision(
+        show_interactions_of_commit, all_unique_hashes
+    )
+
+
 LibraryToHashesMapping = tp.Dict[str, tp.List[str]]
 
 
 def _build_graphviz_edges(
     df: pd.DataFrame,
     graph: Digraph,
     show_edge_weight: bool,
-    edge_weight_threshold: tp.Optional[EdgeWeightThreshold] = None
+    edge_weight_threshold: tp.Optional[EdgeWeightThreshold] = None,
+    show_interactions_of_commit: tp.Optional[str] = None
```

Comment:
```suggestion
    show_only_interactions_of_commit: tp.Optional[str] = None
```
Otherwise, it's not clear from the param that we remove all interactions that don't belong to the commit.